docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>very minor detail, but I think equations would look better without the "*" multiplicative sign.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>